examples/node/src/main.ts

```diff
@@ -1,6 +1,8 @@
-import VIAM = require('@viamrobotics/sdk');
-import wrtc = require('node-datachannel/polyfill');
-import connectNode = require('@connectrpc/connect-node');
+const VIAM = require('@viamrobotics/sdk');
+const wrtc = require('node-datachannel/polyfill');
+const connectNode = require('@connectrpc/connect-node');
+
+// @ts-expect-error
```

Comment:
This is required to pass the TypeScript compiler (`tsc`). If the project was updated to use something like [`tsx`](https://tsx.is/node-enhancement#swap-node-for-tsx), it could run in a single command: `tsx  --env-file=.env src/main.ts`

Otherwise, once Node.js v22 is more widely used, it could be run directly without the extra dependency using the [type stripping feature](https://nodejs.org/en/learn/typescript/run-natively): `node --experimental-strip-types --env-file=.env src/main.ts`